VideoLocker/src/main/java/org/edx/mobile/util/AppStoreUtils.java

```diff
@@ -0,0 +1,60 @@
+package org.edx.mobile.util;
+
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.View;
+import android.widget.Toast;
+
+import org.edx.mobile.BuildConfig;
+import org.edx.mobile.R;
+
+/**
+ * Utility class for interacting with an app store, or the
+ * Play Store specifically.
+ */
+public final class AppStoreUtils {
+    // Make this class non-instantiable
+    private AppStoreUtils() {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Open an app store to display the app.
+     *
+     * @param context A Context for starting the new Activity.
+     */
+    public static void openAppInAppStore(final Context context) {
+        Intent intent = new Intent(Intent.ACTION_VIEW);
+        intent.addCategory(Intent.CATEGORY_BROWSABLE);
+        intent.setData(Uri.parse("market://details?id=" + BuildConfig.APPLICATION_ID));
+        try {
+            context.startActivity(intent);
+        } catch (ActivityNotFoundException e) {
+            intent.setData(Uri.parse("https://play.google.com/store/apps/details?id=" +
+                            BuildConfig.APPLICATION_ID));
```

Comment:
Hmm, so in that case, maybe we should have a list of supported app stores in the config, which we can try to invoke sequentially, and tell the user to install the first one if none of them is available (and there's no web browser installed either). We might also start distributing the app in other app stores in the future, and should be able to open any supported store that the user has already installed on the device.

Note that the Play Store could be specifically opened by just trying to open the HTTP URL of the app in the Play Store website, since it captures those too if it's installed (and other stores shouldn't be capturing it's specific URLs). This also has the advantage of being a generic technique that should work for any app store without having to also check their package names.